alf/examples/ppo_breakout_conf.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import functools
+
+import alf
+from alf.algorithms.agent import Agent
+from alf.networks import ActorDistributionNetwork, ValueNetwork
+
+from alf.examples import atari_conf, ppo_conf
+
+discount = 0.988
```

Comment:
need to config AverageDiscountedReturnMetric"
```python
alf.config(
    "AverageDiscountedReturnMetric",
    discount=discount,
    reward_transformer=reward_transformer)
```

Reply:
Fixed. Thanks!